### ISSUE TITLE
support for parallel testing

### ISSUE THREAD

**Issue by adrian-herscu** [FlaUI/FlaUI#487]
**Describe the bug**
Tried to run several UI tests in parallel. Each test holds its own Application and AutomationBase objects in ThreadLocals.

Getting:

```
System.Runtime.InteropServices.COMException : An event was unable to invoke any of the subscribers (0x80040201)

System.Runtime.InteropServices.COMException : An event was unable to invoke any of the subscribers (0x80040201   at Interop.UIAutomationClient.IUIAutomationTreeWalker.GetFirstChildElement(IUIAutomationElement element)
```

If running sequentially it works.

Any undocumented limitation in FlaUI?

**Comment by Roemer** [comment]
Duplicate of #492
